core/serialisation/data_serialiser.py

```diff
@@ -44,6 +44,24 @@
 )
 
 
+class PostcodeList(fields.Field):
+    """Used to convert between a postcode array (as stored in the DB) and a CSV string (as appears in json/excel)"""
+
+    def _serialize(self, value, attr, obj, **kwargs):
+        if value is None:
+            return ""
+        return ", ".join(value)
+
+    def _deserialize(self, value, attr, data, **kwargs):
```

Comment:
is _deserialize necessary? I can't see that it's actually called anywhere when I run debugger on test_suite? Correct me if I'm wrong but this converts JSON to sqlalchemy, which we don't do?